carts/views.py

```diff
@@ -7,6 +7,37 @@
 from products.models import Option, Product
 from users.utils     import login_required
 
+class CartView(View):
+    @login_required
+    def patch(self, request, cart_id):
+        try:
+            if not Cart.objects.filter(id = cart_id, user = request.user).exists():
+                return JsonResponse({"message":"INVALID_CART_ID"}, status=404)
+ 
+            data = json.loads(request.body)
+
+            item = Cart.objects.get(id = cart_id, user = request.user)
+            item.quantity = int(data['quantity'])
+            item.save()
+            return JsonResponse({
+                "message"  : "SUCCESS", 
+                "QUANTITY" : item.quantity}, status = 201)
```

Comment:
```suggestion
            return JsonResponse({
                "message"  : "SUCCESS", 
                "QUANTITY" : item.quantity}, status = 200)
```
200이 조금 더 절절한 코드 같습니다.